wagtail/contrib/simple_translation/tests/test_wagtail_hooks.py

```diff
@@ -102,3 +103,71 @@ def test_page_listing_more_buttons(self):
             list(page_listing_more_buttons(blog_page, page_perms))[0],
             wagtailadmin_widgets.Button,
         )
+
+
+class TestConstructSyncedPageTreeListHook(Utils):
+    def unpublish_hook(self, pages, action):
+        assert action == "unpublish"
+        assert isinstance(pages, list)
+
+    def missing_hook_action(self, pages, action):
+        assert action == ""
+        assert isinstance(pages, list)
+
+    @override_settings(WAGTAILSIMPLETRANSLATION_SYNC_PAGE_TREE=True)
+    def test_page_tree_sync_on(self):
+        with hooks.register_temporarily(
+            "construct_synced_page_tree_list", self.unpublish_hook
+        ):
+            for fn in hooks.get_hooks("construct_synced_page_tree_list"):
+                response = fn([self.en_homepage], "unpublish")
+                if response:
```

Comment:
so.. this is fine, but best to add a note and another assertion

```
defined_hooks = hooks.get_hooks("construct_synced_page_tree_list")
for fn in defined_hooks:
    # We should have two implementations. 
    # One in simple_tranalation.wagtail_hook and the other registered above
    self.assertEqual(len(defined_hooks), 2)

    response = fn([self.en_homepage], "unpublish")
    if response:
        # We 

        # ...
```

aside from that, I'd change the asserts in `test_` methods to the relevant TestCase ones. e.g. `self.assertIsInstance()`, `self.assertEqual()`, `self.assertTrue()`

Reply:
I added this as a separate test rather than duplicating the assertion over and over. If that's not cool just let me know and I'll change it up. Thanks @zerolab!